packages/facet-filter/src/components/inputs/slider/SliderView.js

```diff
@@ -3,8 +3,24 @@
 /* eslint-disable react/jsx-indent */
 /* eslint-disable object-curly-newline */
 import React, { useEffect, useState } from 'react';
+/*
+import {
+  useLocation,
+  useNavigate,
+} from 'react-router-dom';
+*/
+import { generateQueryStr } from '@bento-core/util';
```

Comment:
Unused import for now


```suggestion
// import { generateQueryStr } from '@bento-core/util';
```

Reply:
good catch!